### PR TITLE
Moved all of the samples into the parent and 

### PR DESCRIPTION
**Pull request by cosbor11**
Also removed the restful web onyx server samples

### CODE REVIEW

**Comment by timbob1000** [comment]
How did onyx-library get re-introduced?  I thought I removed it.  Perhaps I missed it in the parent pom.

File: onyx-database-tests/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
     <modelVersion>4.0.0</modelVersion>
-    <name>onyx-database-tests</name>
+    <name>${project.groupId}:${project.artifactId}</name>
```
Comment (timbob1000) [inline onyx-database-tests/pom.xml]: Should the artifactId be before the name?

File: pom.xml [hunk 2]
```diff
@@ -100,6 +100,7 @@
         <module>onyx-remote-driver</module>
         <module>onyx-library</module>
```
Comment (timbob1000) [inline pom.xml]: onyx-library should not be included since it was removed.

File: onyx-database-tests/pom.xml (hunk 1 above)
Comment (cosbor11) [inline onyx-database-tests/pom.xml]: This is how the Maven Central guide recommends doing it.  See: http://central.sonatype.org/pages/requirements.html

File: pom.xml (hunk 2 above)
Comment (cosbor11) [inline pom.xml]: Not sure why onyx-lib is still in there; will remove

**Comment by cosbor11** [comment]
Approve, Merged with develop branch (caused the ref to onyx-library to disappear)

**Comment by cosbor11** [comment]
Tim!! Please approve this and merge it: https://github.com/OnyxDevTools/onyx-database-parent/pull/12

**Pull request merged**